### PR TITLE
[karmor summary] Colors added(RED,GREEN) for ALLOW/DENY states

### PR DESCRIPTION
**Pull request by yasin-cs-ko-ak**
Signed-off-by:  Mohammed Yasin <yasin@accuknox.com>

### CODE REVIEW

File: summary/table.go [hunk 1]
```diff
@@ -41,7 +42,13 @@ func DisplaySummaryOutput(resp *opb.Response, revDNSLookup bool, requestType str
 				procStrSlice = append(procStrSlice, procData.ProcName)
 				procStrSlice = append(procStrSlice, procData.Count)
 				procStrSlice = append(procStrSlice, procData.UpdatedTime)
-				procStrSlice = append(procStrSlice, procData.Status)
+				agc := ansi.ColorFunc("green")
```
Comment (seswarrajan) [inline summary/table.go:45]: @yasin-cs-ko-ak Please refrain from calling in multiple places. Call in one place and use it here.

**Review by seswarrajan — CHANGES_REQUESTED** [review]

File: go.mod [hunk 2]
```diff
@@ -50,6 +50,7 @@ require (
 require (
 	github.com/cavaliergopher/grab/v3 v3.0.1
 	github.com/google/go-github v17.0.0+incompatible
+	github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d
```
[the remark this answers is not in the document]
Comment (yasindce1998) [reply go.mod:53]: I used that "fatih/color" it's not giving any output. So I used the other one. 
That seems better.

[the remark this answers is not in the document]
Comment (yasin-cs-ko-ak) [reply go.mod:53]: @daemon1024 I tried the "fatih/color" it is not giving me any outputs. So I used the ansi one.

File: summary/table.go (hunk 1 above)
Comment (yasin-cs-ko-ak) [reply summary/table.go]: @seswarrajan Thank you for reviewing. I updated the changes.

**Comment by daemon1024** [comment]
DCO failing for some reason
Can you switch name to your username in the signoff and check?

**Comment by yasin-cs-ko-ak** [comment]
> DCO failing for some reason Can you switch name to your username/git config name in the signoff and check?

Yes. I changed it.

**Comment by daemon1024** [comment]
Still failing :\

**Comment by daemon1024** [comment]
You signing off using git commit -s right?

**Review by daemon1024 — APPROVED** [review]

**Comment by yasin-cs-ko-ak** [comment]
@daemon1024 @seswarrajan Thank you for reviewing :))

**Pull request merged**